restapi/src/main/java/io/stargate/web/resources/RequestHandler.java

```diff
@@ -38,29 +38,31 @@ public static Response handle(Callable<Response> action) {
     } catch (ErrorCodeRuntimeException errorCodeException) {
       return errorCodeException.getResponse();
     } catch (NotFoundException nfe) {
-      logger.info("Resource not found", nfe);
+      logger.info("Resource not found (NotFoundException->NOT_FOUND): {}", nfe.getMessage());
       return Response.status(Response.Status.NOT_FOUND)
           .entity(
               new Error(
                   "Resource not found: " + nfe.getMessage(),
                   Response.Status.NOT_FOUND.getStatusCode()))
           .build();
     } catch (IllegalArgumentException iae) {
-      logger.info("Bad request", iae);
+      logger.info("Bad request (IllegalArgumentException->BAD_REQUEST): {}", iae.getMessage());
       return Response.status(Response.Status.BAD_REQUEST)
           .entity(
               new Error(
                   "Bad request: " + iae.getMessage(), Response.Status.BAD_REQUEST.getStatusCode()))
           .build();
     } catch (InvalidRequestException ire) {
-      logger.info("Bad request", ire);
+      logger.info("Bad request (InvalidRequestException->BAD_REQUEST): {}", ire.getMessage());
```

Comment:
Isn’t the `BAD_REQUEST` redundant?

Reply:
It is; I am not sure what'd be most meaningful combination here. Intent is/was to indicate Exception gotten, HTTP response given, and do that consistently. So I am definitely open to a better scheme.

Reply:
Will merge as-is but can easily improve as a follow-up if we can figure out better scheme here.